htmd/charge/esp.py

```diff
@@ -92,7 +92,7 @@ class MoleculeGrid:
            [ 1.35323883, -3.08583052, -0.17044048],
            [-0.80667116,  0.64144373, -0.99312377],
            [ 2.45040501, -2.23107494, -0.48419202]])
-    >>> grid.writeXYZ('H2O_1_2__50.xyz')
+    >>> #grid.writeXYZ('H2O_1_2__50.xyz')  # Uncomment for visualization
```

Comment:
What do you mean by this comment? Visualization? It's file writing

Reply:
Visual inspection of the molecular grid with VMD.

Reply:
So the issue is that you cannot write in cwd in the container?

Reply:
Yes!

Reply:
Could you try maybe stuff like 2) in this solution? https://stackoverflow.com/a/47362117
It's possible to change the test dir for doctests and I'd prefer that if you want to keep the doctests alive

Reply:
The test are alive, just a files for visualization are not made.